osquery/events/darwin/iokit.h

```diff
@@ -21,13 +21,9 @@
 
 #include "osquery/core/conversions.h"
 
-namespace osquery {
+#include <osquery/core/darwin/iokit.hpp>
```

Comment:
Use double quotes for relative header includes. Also we typically make use of `.h` for C++ headers throughout the repo.

Reply:
This is all existing code, just split into new files.  I chose not to edit the code, since it was a known quantity.  Those edits make sense, will get those in.  I prefer to leave the .hpp extension for this Darwin-specific file to avoid conflict if included in a .mm objectivec++ file.